jobs/JEVS_WAFS_ATMOS_PLOTS

```diff
@@ -15,7 +15,7 @@ export MET_VERSION=`echo $MET_VERSION | sed "s/\([^.]*\.[^.]*\)\..*/\1/g"`
 ############################################
 export SENDECF=${SENDECF:-YES}
 export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
+export SENDDBN=${SENDDBN:-NO}
```

Comment:
@PerryShafran-NOAA If we're using the fake dbnet alerts (see dev driver), should this actually be set to YES? 

CC @YaliMao-NOAA

Reply:
@PerryShafran-NOAA Yes, I just did that. Thank you.